listenbrainz/webserver/views/playlist_api.py

```diff
@@ -860,22 +860,33 @@ def export_playlist(playlist_mbid, service):
     if not is_valid_uuid(playlist_mbid):
         log_raise_400("Provided playlist ID is invalid.")
 
-    if service != "spotify":
+    if service != "spotify" and service != "soundcloud" and service != "apple_music":
         raise APIBadRequest(f"Service {service} is not supported. We currently only support 'spotify'.")
```

Comment:
Text here should be updated to add "and 'apple_music'" at the end
It is worth checking if there are other similar instances in this method where some text needs an update.